Makefile

```diff
@@ -160,15 +160,15 @@ $(SWAGGER_YAML): $(APISERVER_DEPS) | dist
 gen-openapi-client: internal/api/public/client.go ## Generate the OpenAPI Client
 internal/api/public/client.go: internal/docs/swagger.yaml | dist
 	$(ECHO_PREFIX) printf "  %-12s internal/docs/swagger.yaml\n" "[OPENAPI CLIENT GEN]"
-	$(CMD_PREFIX) rm -rf internal/api/public
+	$(CMD_PREFIX) rm -f $(shell find internal/api/public | grep .go | grep -v _custom.go)
```

Comment:
If you want to go hardcore make ... untested and mostly for fun because I can:

```
$(filter-out %_custom.go,$(filter %.go,$(wildcard internal/api/public/*)))
```

Reply:
My make foo is weaker than my shell foo.